tests/MyGame/Example/Monster.py

```diff
@@ -131,8 +131,8 @@ def TestarrayofstringLength(self):
             return self._tab.VectorLen(o)
         return 0
 
-# /// an example documentation comment: this will end up in the generated code
-# /// multiline too
+    # an example documentation comment: this will end up in the generated code
```

Comment:
Move down immediately after `# Monster`?

Reply:
Not sure what you're asking.. as long as the indentation is fixed.

Reply:
User-defined comments add before the getter's comment:
```python
    # an example documentation comment: this will end up in the generated code
    # multiline too
    # Monster
    def Testarrayoftables(self, j):
```

Another way, generate:
```python
    # Monster
    # an example documentation comment: this will end up in the generated code
    # multiline too
    def Testarrayoftables(self, j):
```

Or use docstring from PEP:
```python
    # Monster
    def Testarrayoftables(self, j):
        """an example documentation comment: this will end up in the generated code
        multiline too
        """
```